inc/customizer.php

```diff
@@ -209,86 +209,87 @@ function zillah_customize_register( $wp_customize ) {
 	/* Google fonts  */
 	$wp_customize->add_setting(
 		'zillah_google_fonts_one', array(
-			'default' => 0,
+			'default'           => 0,
 			'sanitize_callback' => 'sanitize_text_field',
 		)
 	);
 
 	$wp_customize->add_control(
 		new Zillah_Google_Fonts_Control(
 			$wp_customize, 'zillah_google_fonts_one', array(
-				'label'    => 'Select first font family ( content )',
-				'section'  => 'zillah_home_theme_option_section',
-				'priority' => 4,
+				'label'           => 'Select first font family ( content )',
+				'section'         => 'zillah_home_theme_option_section',
+				'priority'        => 4,
 				'ti_google_fonts' => array(
 					array(
 						'font_family' => 'Merriweather',
-						'type' => 'serif',
-						'subset' => '400',
+						'type'        => 'serif',
+						'subset'      => '400',
 					),
 					array(
 						'font_family' => 'Open Sans',
-						'type' => 'sans-serif',
-						'subset' => '400',
+						'type'        => 'sans-serif',
+						'subset'      => '400',
 					),
 					array(
 						'font_family' => 'Josefin Slab',
-						'type' => 'serif',
-						'subset' => '400',
+						'type'        => 'serif',
+						'subset'      => '400',
 					),
 					array(
 						'font_family' => 'Ubuntu',
-						'type' => 'sans-serif',
-						'subset' => '400',
+						'type'        => 'sans-serif',
+						'subset'      => '400',
 					),
 					array(
 						'font_family' => 'Vollkorn',
-						'type' => 'serif',
-						'subset' => '400',
+						'type'        => 'serif',
+						'subset'      => '400',
 					),
+					apply_filters( 'zillah_extend_fonts_one', false ),
```

Comment:
You should apply this filter on the whole array, not just on an element. After that you can filter it in the child theme by doing an array push to add the new font. 👍

Reply:
Also I'd rename it to something like `zillah_filter_body_fonts`.